state/futures/types.ts

```diff
@@ -64,6 +64,9 @@ export type FuturesTransactionType =
 	| 'withdraw_isolated'
 	| 'modify_isolated'
 	| 'close_isolated'
+	| 'close_cross_margin'
+	| 'cancelDelayed_isolated'
```

Comment:
could we change to `cancel_delayed_isolated` and `execute_delayed_isolated` to keep consistent formatting